### DISCUSSION TITLE
Self hosted Version

### DISCUSSION THREAD

**Discussion by OscarWilde** [codex-team/editor.js#2647] [Q&A] [answered]
I need to host the scripts on my own Site, but where do I find the latest compiled Version of editorjs.js and the corresponding files? In the Release is no dist Folder.

**Comment by neSpecc** [comment, marked as answer]
Releases contain bundles: `editorjs.mjs` and `editorjs.umd.js`. You can host them

**Reply by OscarWilde** [reply]
Thanks for your answer! No additional CSS Files needed?

**Reply by OscarWilde** [reply]
Where is the difference between the files?

**Reply by neSpecc** [reply]
Correct. They are included

**Reply by neSpecc** [reply]
`.mjs` is a ES module bundle. Use it when you're importing it in esm package via `import`.
`.umd.js` could be used to import file using a `<script src="">` in html page. Or to import it in Common JS project

**Reply by OscarWilde** [reply]
Thanks a lot!

**Reply by OscarWilde** [reply]
Can I ask you another Question? How do I include more modules, like Header or Image?
